src/styles/actor/_index.scss

```diff
@@ -202,6 +202,7 @@
         .sheet-body {
             grid-area: content;
             position: relative;
+            z-index: 0;
```

Comment:
```suggestion
```

And replace in `_spell-collection.scss`:
```diff
     &[data-category="spontaneous"] {
         .virtual {
-            position: relative;
-            &::before {
-                background-color: #0005ff1d;
-                content: "";
-                height: 100%;
-                mix-blend-mode: saturation;
-                pointer-events: none;
-                position: absolute;
-                width: 100%;
-            }
+            backdrop-filter: blur(8px) hue-rotate(210deg) saturate(2);
         }
```

Reply:
Oh yeah, that looks nicer.